en/docs/admin-portal/applications/overview.md

```diff
@@ -0,0 +1,9 @@
+# Overview
+
+## Add a new Application
+
+## Search an application
+
+## Edit application
+
+## Delete application
```

Comment:
Since this page is just a list of the child pages, do we really need this page?

Reply:
This page contains instructions on how to use different buttons in the list view.

Reply:
Better to have this with a simple description. Otherwise, it's hard to find one document with all the operations for 'Application Management' as well as the landing page for 'Applications' heading. when sharing links with external parties this is helpful